src/tests/test_data.py

```diff
@@ -51,4 +52,42 @@ def get_other_rec_test_models():
                     Sound(note=None,
                           duration=constants.RHYTMIC_VALUE_TO_DURATION[spl[1]])
                 )
+    current_testcase.sounds = currect_sounds
+    result.append(current_testcase)
+    return result
+
+
+def get_carol_test_models():
```

Comment:
DRY?
This is almost the same as `get_other_rec_test_models`

Reply:
I created _get_test_models(test_data_file) and moved duplicate code there